app/assets/javascripts/discourse/app/components/modal/create-account.js

```diff
@@ -215,8 +217,23 @@ export default class CreateAccount extends Component.extend(
     });
   }
 
+  @action
+  showPasswordValidation() {
+    if (this.passwordValidation.reason) {
+      this.set("passwordValidationVisible", true);
+    } else {
+      this.set("passwordValidationVisible", false);
+    }
```

Comment:
code ⛳ you could also do: 

```suggestion
    this.set("passwordValidationVisible", Boolean(this.passwordValidation.reason));
```

Reply:
oc, nice! :ok_hand: